float64 cameraDistance
float64 cameraYaw
float64 cameraPitch
float64[] cameraTargetPosition